style.css

```diff
@@ -0,0 +1,114 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+body {
+  display: flex;
+  flex-direction: column;
+  background-color: #cac4c4;
```

Comment:
- [ ] Please, to make sure that the **body** of the page has the same background colour compared to the template, I recommend using this one `#E5E5E5` instead. :ok_hand: 😊